gateway-js/src/IntrospectAndCompose/loadServicesFromRemoteEndpoint.ts

```diff
@@ -1,12 +1,12 @@
 import { GraphQLRequest } from 'apollo-server-types';
```

Comment:
Not a new issue in this PR, but the function exported from this file doesn't match the file name.

Reply:
3b03867f